GeneratorInterface/GenFilters/interface/PythiaFilter.h

```diff
@@ -39,6 +39,10 @@ namespace edm {
   class HepMCProduct;
 }
 
+namespace HepMC {
```

Comment:
hi @zc11 - you need to include the header for class FourVector since you use it just below.